desktop/apps/about/client/index.coffee

```diff
@@ -1,5 +1,5 @@
 Backbone = require 'backbone'
-AboutRouter = require './router.coffee'
+AboutRouter = require './router'
```

Comment:
Thanks! Removing the need for .coffee extensions is something I've wanted to do for a while. I assume this works in development for client-side code too? If not one can set it to be supported [in browserify](https://github.com/substack/node-browserify#browserifyfiles--opts).